src/classes/MAPR_PrimaryAffiliations_TEST.cls

```diff
@@ -574,11 +599,135 @@ private class MAPR_PrimaryAffiliations_TEST {
+    * @description Test to ensure that getModelFromMapping returns a valid model with an error message indicating an invalid record type 
+    * mapping when validation is on, an invalid record type is specified and a valid field that does lookup to Account is mapped.
+    **************************************************************************************************************************************/ 
+    private static void getModelFromMappingInvalidRTAndLookupFieldValidationOn(){
+        MAPR_PrimaryAffiliations_TEST.insertAfflMappingsWithInaccurateRecordTypeNameAndAccuratePrimaryFieldLabels();
+        MAPR_PrimaryAffiliations maprPrimaryAfflInstance = new STUB_PrimaryAffiliationValidationOn();
+
+        List<Affl_Mappings__c> primaryAfflMappings = UTIL_CustomSettingsFacade.getPrimaryAfflMappings();
+
+        Test.startTest();
+            MODL_PrimaryAfflMapper primaryAfflModel = maprPrimaryAfflInstance.getModelFromMapping(primaryAfflMappings[0]);
+        Test.stopTest();
+
+        String prefix = UTIL_Namespace.StrTokenNSPrefix('');
+
+        System.assert(primaryAfflModel != null);
+        System.assertEquals(false, primaryAfflModel.getIsValid());
+        System.assertEquals(MAPR_PrimaryAffiliations_TEST.getExpectedInvalidRTMappingErrorMessage(primaryAfflMappings[0]), primaryAfflModel.getInvalidErrorMessage());
+        System.assertEquals('Business Org test', primaryAfflModel.getAccountRecordTypeId());
+        System.assertEquals(prefix + 'Primary_Organization__c', primaryAfflModel.getContactFieldApiName());
+        System.assertEquals('Primary_Organization__c', primaryAfflModel.getSettingsContactFieldValue());
+        System.assertEquals('Business Org test', primaryAfflModel.getSettingsRecordTypeFieldValue());
+    }
+
+    /**************************************************************************************************************************************
+    * @description Test to ensure that getModelFromMapping returns a valid model with an error message indicating an invalid lookup field 
+    * mapping when validation is on, a valid record type is specified and a valid field that does not lookup to Account is mapped.
+    **************************************************************************************************************************************/ 
+    private static void getModelFromMappingValidRTAndNonLookupFieldValidationOn(){
+        MAPR_PrimaryAffiliations_TEST.insertAfflMappingsWithValidRecordTypeNameAndNonLookupPrimaryFieldLabels();
+        MAPR_PrimaryAffiliations maprPrimaryAfflInstance = new STUB_PrimaryAffiliationValidationOn();
+
+        List<Affl_Mappings__c> primaryAfflMappings = UTIL_CustomSettingsFacade.getPrimaryAfflMappings();
+
+        Test.startTest();
+            MODL_PrimaryAfflMapper primaryAfflModel = maprPrimaryAfflInstance.getModelFromMapping(primaryAfflMappings[0]);
+        Test.stopTest();
+
+        String prefix = UTIL_Namespace.StrTokenNSPrefix('');
```

Comment:
Another `UTIL_Namespace.StrTokenNSPrefix(String)` on the specific String in question.